### ISSUE TITLE
Improve user experience of tooltip sorting

### ISSUE THREAD

**Issue by ivan-aksamentov** [hodcroftlab/covariants#104]
Followup of #99 

Currently per-variant page contains a dropdown and a checkbox which both control the sorting options of the plot tooltips.

This is a cumbersome design choice from my side. I now think that a better user experience would be to have only dropdown with 4 options: {freq, coutry} x {ascending, descending}, similar to what many shopping websites have for example.

**Comment by emmahodcroft** [comment]
I don't have strong feelings but I am happy to move the design in that direction. As you say, it's common on shopping sites.

**Comment by emmahodcroft** [comment]
I think we have this too now?

**Comment by ivan-aksamentov** [comment]
No, my proposal here is to replace a dropdown+checkbox with only dropdown containing all possible permutations. We still have dropdown+checkbox currently.

**Comment by emmahodcroft** [comment]
Ah ok - good thing I checked! I'll leave it open then.
